sdk/storage/inc/common/common_headers_request_policy.hpp

```diff
@@ -12,7 +12,10 @@ namespace Azure { namespace Storage {
     explicit CommonHeadersRequestPolicy() {}
     ~CommonHeadersRequestPolicy() override {}
 
-    HttpPolicy* Clone() const override { return new CommonHeadersRequestPolicy(*this); }
+    std::unique_ptr<HttpPolicy> Clone() const override
```

Comment:
What does this CommonHeadersRequestPolicy class even do? 
There is no comment explaining it. Why do we need this class?

Reply:
It's used to add headers that apply to every HTTP request. But for the moment, I guess there's no other headers besides `x-ms-date`. So I guess we should rename it to `DatePolicy`?

Reply:
Not doing anything in this PR, as there is a clarification, and it is not clear whether we will need to add new headers, in which case we may need to do the rename again, please reactivate this comment if you think we should rename it now.